_posts/2020-12-09-cockpit-234.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Cockpit 234 and Cockpit-Podman 26
+author: mvo
+date: 2020-12-09
+tags: cockpit linux
+slug: cockpit-234
+category: release
+summary: Cockpit with various bug fixes and performance improvements
+---
+
+Cockpit is the [modern Linux admin
+interface](https://cockpit-project.org/).  We release regularly. Here
+are the release notes from Cockpit version 234 and Cockpit-Podman 26.
+
+## Machines: Allow editing VM's CPU mode and model
+
+When changing the CPU model, the goals are to maximise performance, while protecting guest OS against various CPU hardware flaws, and optionally enabling live migration between hosts with heterogeneous CPU models.
+
+There are three different options supported.
+* Host passthrough
+
+This passes the host CPU model features, model, stepping, exactly to the guest.
+
+* Host model
+
+This uses the QEMU `Named model` feature, automatically picking a CPU model that is similar the host CPU, and then adding extra features to approximate the host model as closely as possible. This does not guarantee the CPU family, stepping, etc will precisely match the host CPU, as they would with `Host passthrough`, but gives much of the benefit of passthrough, while making live migration safe.
+
+* Custom / Named model
+
+QEMU comes with a number of predefined named CPU models, that typically refer to specific generations of hardware released by Intel and AMD. These allow the guest VMs to have a degree of isolation from the host CPU, allowing greater flexibility in live migrating between hosts with differing hardware.
+
+More information about this feature can be found in the [QEMU documentation page](https://www.qemu.org/docs/master/system/qemu-cpu-models.html).
+
+![screenshot of the cpu mode configuration](/images/machines-cpu-model.png)
+
+## Machines: Add support for cloning VMs
+
+Cloning creates a new VM that uses its own disk image for storage, but most of the clone’s configuration and stored data is identical to the source VM. This makes it possible to prepare a number of VMs optimized for a certain task without the need to optimize each VM individually.
+
+![screenshot of the machine cloning](/images/machines-cloning.png)
+
+## The dashboard has been removed
+
+The very limited functionality of the dashboard does not justify its
+continued maintenance, in our opinion.  The Shell can connect to
+remote hosts and Grafana etc are much better at monitoring multiple
+machines.
```

Comment:
Due to it's limited functionality the dashboard page has been removed. Additional hosts can be managed through shell without any need for a separate package. The graphs part was not providing enough information to be kept as for detailed metrics a separate page is available on the overview page and for multihost monitoring other tools (e.g. Grafana) provide more useful data.

Reply:
:+1: